exec.go

```diff
@@ -41,42 +53,68 @@ var execCommand = cli.Command{
 	},
 }
 
-func execProcess(context *cli.Context, config *specs.Process) (int, error) {
+func execProcess(context *cli.Context, args []string) (int, error) {
 	container, err := getContainer(context)
 	if err != nil {
 		return -1, err
 	}
-	process := newProcess(*config)
 
+	bundle := container.Config().Rootfs
+	if err := os.Chdir(path.Dir(bundle)); err != nil {
+		return -1, err
+	}
+	spec, _, err := loadSpec(specConfig, runtimeConfig)
+	if err != nil {
+		return -1, err
+	}
+
+	// override the cwd, if passed
+	if context.String("cwd") != "" {
+		spec.Process.Cwd = context.String("cwd")
+	}
+
+	// append the passed env variables
+	for _, e := range context.StringSlice("env") {
+		spec.Process.Env = append(spec.Process.Env, e)
+	}
+
+	// set the tty
+	spec.Process.Terminal = context.Bool("tty")
```

Comment:
if tty isn't specified on the exec cmd won't this change what was used in the config.json?